src/main/scala/com/neu/edu/FlightPricePrediction/db/MinioOps.scala

```diff
@@ -62,27 +69,38 @@ object MinioOps {
   def delete(bucket: String, id: String): Unit = {
     // remove object
     minioClient.removeObject(bucket, id)
-
-    // get size of the bucket
-    println(List(minioClient.listObjects(bucket)).size)
   }
 
   def putFile(bucket: String, id: String, filePath: String) = {
-    Try {
-      put(bucket, id, Files.readAllBytes(Paths.get(filePath)))
+    try Success(put(bucket, id, Files.readAllBytes(Paths.get(filePath)))) catch {
+      case NonFatal(e)=>Failure(MinioFileException(filePath,e))
```

Comment:
Before you catch and then throw an exception, remember to   do some logging, so that we can find where the problem is